index.html

```diff
@@ -1,19 +1,159 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>AlterTime: Mage Class Hall</title>
+
+    <!-- fontawesome Cdn Link 6.2.1 -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
 
     <!-- Custom css file link -->
     <link rel="stylesheet" href="css/style.css">
 
     <!-- Custom JS file link -->
     <script defer src="js/main.js"></script>
-    
 </head>
+
 <body>
-    
+    <!-- Navbar Section Starts here -->
+    <section id="navbar">
+        <nav class="sub__navbar">
+            <div class="socialIcon">
+                <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            </div>
+            <div class="sub__navMenu">
+                <ul class="navLink">
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+            </div>
+        </nav>
+        <nav class="main__navbar">
+            <a href="#" class="logo"><img width="60" src="img/mage icon.png"
+                    alt="logo"></a>
+            <div class="main__navMenu">
+                <ul class="navLink">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#Program">Program</a></li>
+                    <li><a href="#Join">Join</a></li>
+                    <li><a href="#partner">Sponsor</a></li>
+                    <li><a href="#News">News</a></li>
```

Comment:
- [ ] Kindly add a link to the home page, so that users can click the link and return to the home page😉